.eslintrc.js

```diff
@@ -299,6 +299,16 @@ module.exports = {
         'no-console': 'off',
       },
     },
+    // demos - proptype generation
+    {
+      files: ['docs/data/base/components/modal/UseModal.js'],
```

Comment:
Maybe to generalize, if it's code generated, I guess this applies to everything.